l1-contracts/src/core/libraries/rollup/EpochProofLib.sol

```diff
@@ -45,6 +43,28 @@ library EpochProofLib {
   // such as sacrificial hearts, during rituals performed within temples.
   address public constant CUAUHXICALLI = address(bytes20("CUAUHXICALLI"));
 
+  /**
+   * @notice  Submit a proof for an epoch in the pending chain
+   *
+   * @dev     Will emit `L2ProofVerified` if the proof is valid
```

Comment:
Might be worth saying something about pruning here?

Reply:
Were thinking that it was fine as the first line in the code was prune if possible 🤷

Reply:
yeah probably true